<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Replace</class>
 <widget class="QWidget" name="Replace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mismatch detected in keyfiles.</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="toolTip">
      <string>Choose the version of the keyfile to retain</string>
     </property>
     <property name="text">
      <string>Please select the keyfile version you want to keep:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QRadioButton" name="dbRadioButton">
       <property name="toolTip">
        <string>Keep the keyfile stored in the database</string>
       </property>
       <property name="text">
        <string>Version in Database</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="diskRadioButton">
       <property name="toolTip">
        <string>Keep the keyfile stored on disk</string>
       </property>
       <property name="text">
        <string>Version in Disk</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTreeView" name="dbTreeView">
       <property name="toolTip">
        <string>Keyfile content from the database</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTreeView" name="diskTreeView">
       <property name="toolTip">
        <string>Keyfile content from the disk</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="toolTip">
      <string>Details of the selected file</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalButtonlLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="toolTip">
        <string>Save the selected keyfile version</string>
       </property>
       <property name="text">
        <string>Proceed</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
